org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.xtend

```diff
@@ -134,7 +143,7 @@ import org.eclipse.xtext.validation.Issue
 		]
 
 		requestManager.runWrite [ cancelIndicator |
-			val rootURI = URI.createFileURI(params.rootPath).toPath.toUri
+			val rootURI = URI.createURI(rootPath.toUri.toString)
```

Comment:
It would be nice to have tests for it if there are not yet. Could you try to break it and see whether some tests break :)